package.json

```diff
@@ -78,6 +83,7 @@
     "targz": "1.0.1",
     "uglify-js": "2.6.4",
     "underscore": "1.8.3",
-    "watch": "0.19.1"
+    "watch": "0.19.1",
+    "webpack": "1.14.0"
```

Comment:
If it's out of beta I 👍 on this, unless it requires a significant amount of work so that we want to keep it to the next PR